dogfood/contents/main.tf

```diff
@@ -290,6 +290,18 @@ resource "coder_agent" "dev" {
     timeout      = 5
   }
 
+  resources_monitoring {
+    memory {
+      enabled = true
+      threshold = 80
```

Comment:
Idea: let's allow folks to specify their own thresholds via parameters because 80% might not work for everyone.